common-app-lib/src/cmr/common_app/data/search/collection_for_gran_acls_caches.clj

```diff
@@ -11,7 +11,7 @@
   [cmr.common.log :as log :refer (debug info warn error)]
   [clojure.walk :as walk]
   [cmr.common-app.services.search.query-execution :as qe]
-  [cmr.common-app.services.search.query-model :as q-mod]
+  [cmr.common-app.services.search.query-model :as qm]
```

Comment:
we have been trending in the direction of not using two letter :as values especially when they are so close to another.

Reply:
fixed